test/torchaudio_unittest/functional/functional_impl.py

```diff
@@ -582,6 +583,51 @@ def test_rnnt_loss_costs_and_gradients_random_data_with_numpy_fp32(self):
             ref_costs, ref_gradients = rnnt_utils.compute_with_numpy_transducer(data=data)
             self._test_costs_and_gradients(data=data, ref_costs=ref_costs, ref_gradients=ref_gradients)
 
+    @parameterized.expand(
+        [
+            (1,),
+            (2,),
+            (3,),
+        ]
+    )
+    def test_rtf_power(self, iterations):
```

Comment:
Even though it is simple, please add the docstring. So that random future maintainers get what this test is about, without reading the code. (the same goes to the other test)